react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -668,18 +900,21 @@ function _mapStateToProps(state) {
     const toolbarButtons = getToolbarButtons(state);
     const { testing = {}, iAmRecorder } = state['features/base/config'];
     const enableThumbnailReordering = testing.enableThumbnailReordering ?? true;
-    const { visible, remoteParticipants } = state['features/filmstrip'];
+    const { visible, remoteParticipants, width: verticalFilmstripWidth } = state['features/filmstrip'];
     const reduceHeight = state['features/toolbox'].visible && toolbarButtons.length;
     const remoteVideosVisible = shouldRemoteVideosBeVisible(state);
     const { isOpen: shiftRight } = state['features/chat'];
     const {
-        gridDimensions = {},
+        gridDimensions: dimensions = {},
         filmstripHeight,
         filmstripWidth,
         thumbnailSize: tileViewThumbnailSize
     } = state['features/filmstrip'].tileViewDimensions;
     const _currentLayout = getCurrentLayout(state);
     const disableSelfView = shouldHideSelfView(state);
+    const _resizableFilmstrip = isFilmstripResizable(state);
+    const _verticalViewGrid = _resizableFilmstrip && verticalFilmstripWidth.current > FILMSTRIP_GRID_BREAKPOINT;
```

Comment:
I saw the same computations in the actions. Maybe we should move this to a function in order to keep them consistent?